src/ImageRunModal.scss

```diff
@@ -0,0 +1,18 @@
+@import '@patternfly/patternfly/layouts/Grid/grid.css';
+
+// Limit the max-width more the override in podman.scss which is 40%
+#run-image-dialog-memory, #run-image-cpu-priority {
+     max-width: 15%;
+}
+
+#memory-unit-select {
+     max-width: 20%;
+}
+
+#run-image-dialog-memory-limit-checkbox, #run-image-dialog-cpu-priority-checkbox {
+     margin-top: 12px;
```

Comment:
This is suboptimal as the patternfly spacing is different. We come up with a better solution for this.

Reply:
For spacing, we should try to stick to variables like var(--pf-global--spacer--md), but is this trying to fix alignment or something? (In that case, we should just align it.)

I'll take a look.

Reply:
Use rems not percentage. The percentage works horribly in mobile mode, where you want the element to all of the space (or for sure more)

Reply:
This is resolved in a new push. Percentages are replaced with `rem` and the checkbox margin is replaced with a patternfly css class.